src/frontend/js/components/CourseProductCourseRuns/EnrolledCourseRun.tsx

```diff
@@ -71,22 +90,37 @@ const EnrolledCourseRun = ({ enrollment }: Props) => {
           </em>
         </div>
         {states.error && <p className="course-runs-list__errors-feedback ">{states.error}</p>}
+
+        {!isStarded && (
+          <p className="course-runs-list__errors-feedback ">
+            <FormattedMessage {...messages.courseRunStartIn} values={{ relativeStartDate }} />
+          </p>
+        )}
         <div className="course-runs-item">
-          <a
-            href={enrollment.course_run.resource_link}
-            className="course-runs-item__cta button--primary button--pill button--tiny"
-          >
-            <FormattedMessage {...messages.goToCourse} />
-          </a>
-          <button className="button--tiny" onClick={unroll}>
+          {isStarded && (
+            <a
+              href={enrollment.course_run.resource_link}
+              className="course-runs-item__cta button--primary button--pill button--tiny"
+            >
+              <FormattedMessage {...messages.goToCourse} />
+            </a>
+          )}
+
+          {!isStarded && (
+            <div className="course-runs-item__cta button--primary button--pill button--tiny">
```

Comment:
I'll use a `p` tag then I'll add a `disabled` class because currently user can believe this element is call to action.